osm2lanes/Cargo.toml

```diff
@@ -8,6 +8,7 @@ abstutil = { git = "https://github.com/a-b-street/abstreet" }
 anyhow = { workspace = true }
 enumset = { version = "1.0.12", features=["serde"] }
 geom = { workspace = true }
+muv-osm = { version = "0.1.0", path = "../../muv8/muv-osm", features = ["lanes", "serde"] }
```

Comment:
This'll need to be `git = "https://gitlab.com/LeLuxNet/Muv/"` eventually, until things are on crates.io. (I publish very few things to crates.io since the API is still in such constant flux)

Reply:
Exactly, that was my plan. It's just for the time being as I develop it locally. Not sure if cargo has something similar to Go's `replace` feature.